src/app/modules/shared-components/components/add-content/add-content.component.ts

```diff
@@ -8,6 +8,7 @@ export interface AddedContent<T> {
   id?: string,
   title: string,
   type: T,
+  scoreWeight?: number,
```

Comment:
This component has nothing to do with the weight. It is also used for groups where there is no weight. Weight should just be added with the data is received, so in `addChild` of item-children-edit.

Reply:
Fixed